### ISSUE TITLE
USDT to trx transaction revert at trc20 .Help?

### ISSUE THREAD

**Issue by aarohbaba** [tronprotocol/java-tron#4099]
### 1. What did you do?

6acddf10e0bc91b5e9b568df83cbfd883184c5d533c78036c1b92d78941a7e93
I swap USDT to Trx ,USDT got deducted but Trx didn't receive. Kindly help.

### 2. What did you expect to see?

6000 trx in exchange of 589 USDT

### 3. What did you see instead?

589 USDT disappeared but 6000trx shows failed

**Comment by aarohbaba** [comment]
This is wrong, I didn't give any such approval. I simply Swap 589 USDT into Tron there must be some kind of scam happening that takes USDT
![Uploading Screenshot_20211012-120929_TronLink Pro.jpg…]()

**Comment by Benson0224** [comment]
from the transaction history of your account. it shows that your USDT is transferred to TBTZtraavCneBB8TPTYqrm9gffPt7YJcXY before you do the swap. 
that is why your swap failed because the USDT in your account is not enough for the SWAP. 
![Xnip2021-10-12_15-24-34](https://user-images.githubusercontent.com/61768920/136910683-c60310b0-d369-4ce4-a3f3-475ff97971ca.jpg)
